### PR TITLE
CI: take check name into consideration when set

### PR DESCRIPTION
**Pull request by dberzano**

### CODE REVIEW

**Comment by ktf** [comment]
As privately discussed, maybe the best option is to simply rename STATE_CONTEXT to CHECK_NAME, adding a default for it. AFAICT no one depends on `$STATE_CONTEXT`.

File: ci/continuous-builder.sh [hunk 1]
```diff
@@ -18,7 +18,9 @@ export ALIBOT_ANALYTICS_APP_NAME="continuous-builder.sh"
 MIRROR=${MIRROR:-/build/mirror}
 PACKAGE=${PACKAGE:-AliPhysics}
 
-STATE_CONTEXT=build/$PACKAGE${ALIBUILD_DEFAULTS:+/$ALIBUILD_DEFAULTS}
+# This is the check name. If CHECK_NAME is in the environment, use it. Otherwise
+# default to, e.g., build/AliRoot/release (build/<Package>/<Defaults>)
+STATE_CONTEXT=${CHECK_NAME:=build/$PACKAGE${ALIBUILD_DEFAULTS:+/$ALIBUILD_DEFAULTS}}
```
Comment (ktf) [inline ci/continuous-builder.sh]: As privately discussed, maybe the best option is to simply rename STATE_CONTEXT to CHECK_NAME, adding a default for it. AFAICT no one depends on $STATE_CONTEXT.

Comment (dberzano) [inline ci/continuous-builder.sh]: OK thx update is coming

File: ci/continuous-builder.sh [hunk 2]
```diff
@@ -47,7 +49,7 @@ while true; do
   done
 
   if [[ "$PR_REPO" != "" ]]; then
-    HASHES=`list-branch-pr --show-main-branch ${CHECK_NAME:+--check-name $CHECK_NAME} ${TRUST_COLLABORATORS:+--trust-collaborators} ${TRUSTED_USERS:+--trusted $TRUSTED_USERS} $PR_REPO@$PR_BRANCH ${WORKERS_POOL_SIZE:+--workers-pool-size $WORKERS_POOL_SIZE} ${WORKER_INDEX:+--worker-index $WORKER_INDEX} ${DELAY:+--max-wait $DELAY} || report-analytics exception --desc "list-branch-pr failed"`
+    HASHES=`list-branch-pr --show-main-branch --check-name $CHECK_NAME ${TRUST_COLLABORATORS:+--trust-collaborators} ${TRUSTED_USERS:+--trusted $TRUSTED_USERS} $PR_REPO@$PR_BRANCH ${WORKERS_POOL_SIZE:+--workers-pool-size $WORKERS_POOL_SIZE} ${WORKER_INDEX:+--worker-index $WORKER_INDEX} ${DELAY:+--max-wait $DELAY} || report-analytics exception --desc "list-branch-pr failed"`
```
Comment (dberzano) [inline ci/continuous-builder.sh]: Note that now `CHECK_NAME` is always set.

**Comment by dberzano** [comment]
@ktf can you check now please? Thanks!

**Comment by dberzano** [comment]
@ktf just to be sure, you said this can be merged now, right?

**Pull request merged**